tao_compiler/mlir/xla/ral/context/init_stream_executor.cc

```diff
@@ -86,7 +86,8 @@ se::Stream* GetOrCreateDefaultCudaStreamExecutorStream(int device_ordinal,
     // StreamExecutor use cuda primary context.
     // A non-primary context was not verfied by StreamExectuor.
     // So we would not support non-primary cuda context currently.
-    e.stream = new se::Stream(executor, new CUDAStream(cuda_executor, stream));
+    e.stream = new se::Stream(
+        executor, std::make_unique<CUDAStream>(cuda_executor, stream), false);
```

Comment:
See the comment in line 85, unique-ptr may be not safe?

Reply:
The new se::Stream only accepts unique-ptr after rebasing the community TF. When e.stream is not destroyed, the created unique-ptr held by e.stream is safe.